route-rs-runtime/src/link/async_link.rs

```diff
@@ -214,56 +269,105 @@ impl<Packet: Sized> Stream for AsyncEgressor<Packet> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::element::{AsyncIdentityElement, DropElement, IdentityElement, TransformElement};
+    use crate::element::{DropElement, IdentityElement, TransformElement};
     use crate::link::sync_link::SyncLinkBuilder;
-    use crate::link::{ElementLinkBuilder, LinkBuilder};
+    use crate::link::{ElementLinkBuilder, LinkBuilder, TokioRunnable};
     use crate::utils::test::packet_collectors::ExhaustiveCollector;
     use crate::utils::test::packet_generators::{immediate_stream, PacketIntervalGenerator};
     use core::time;
     use futures::future::lazy;
 
+    fn run_tokio(runnables: Vec<TokioRunnable>) {
```

Comment:
I'd like to consolidate these kinds of functions into test helpers, but I can do that after this PR is merged.

Reply:
I was thinking the same thing when I kept copying it lol